project/src/main/webapp/script2.js

```diff
@@ -0,0 +1,9 @@
+function listPollingLocations() {
```

Comment:
Add the copyright information and JavaScript version like in our other source files: https://github.com/googleinterns/voter-central/blob/Travis_Dev2/project/src/main/webapp/candidate-page/candidate-page.component.js#L16